code/index.html

```diff
@@ -4,27 +4,127 @@
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Imbue:wght@500&family=PT+Sans+Narrow:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+<!---------1. HEADER, LOGO, NAVIGATION---------->
+    <header class="header">
+      <img class="logo" src="./img/logo.png" alt="logotype">
+      <nav class="navigation">
+        <a href="#">
+        <div class="navigation-text">About us</div>
+      </a>
+      <a href="#"> 
+      <div class="navigation-text">Join us</div>
+      </a> 
+      <a href="#">    
+      <div class="navigation-text">Schedule</div>
+    </a>
+      </nav>
+    </header>
 
-    <!-- Signup form -->
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<!------ 2. HERO IMG AND TEXT-------->
+      <div class="hero"></div>
+        <picture>
+          <source srcset="./img/battleropes.jpg" media="(max-width: 767px)" />
+          <img class="hero-image" src="./img/highfive.jpg" alt="Hero image" />
+        </picture>
+        <h1 class="hero-text">"Strong people don't put others down, they lift them up"</h1>
+      </div>
+    
+    
+<!-------- 3. INTRO TEXT------->
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <h2 class="intro-text">Looking for a fun and varied type of training? Where you will learn new skills and build up both your strenght, condition as well as your self confidense.</h2> 
+    <p class="smaller-intro-text"">
+      Crossfit is training for life. You will lift, push, pull, jump, run - which all are things you will come accross in your ordinary life. In our box we train together, push each other and celebrate each others achievements. One of the best things about Crossfit is our community which is achived by awesome coaches and even more awesome members. Join us for the fun, the workouts and for the challanges! 
+    </p>
+
+
+    <!------ 4. SIGN UP FORM----->
+<div class="signup-container">
+    <form action="https://httpbin.org/anything" method="POST" class="form">
+        <p class="form-text"> Contact us for a free consultation!</p>
+        <!--NAME,EMAIL-->
+        <label for="first-name">First Name &nbsp &nbsp </label>
+        <input type="text" id="first-name" name="first-name" style="width: 150px;" placeholder="Your first name" required>
+        <br><br>
+        <label for="last-name">Last Name &nbsp &nbsp</label>
+        <input type="text" id="last-name" name="last-name" style="width: 150px;" placeholder="Your last name" required>
+        <br><br>
+        <label for="email">Email adress</label>
+        <input type="email" id="email" name="email" style="width: 150px;" placeholder="Your email" required>
+        <br><br>
```

Comment:
Nice structure! Try to omit the `br`s and use CSS styling to achieve the same thing with e.g. margin